Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:15
+
+MAINTAINER Philipp Hackner <philipp.hackner@protonmail.com>
+
+ADD backend/target/musicompass.jar app.jar
+
+CMD ["sh", "-c", "java -Dserver.port=$PORT -Ddiscogs.auth.discogsToken=$DISCOGS_TOKEN -jar /app.jar"]
```

Comment:
if you are passing spring specific environment variables it might sense to have them start with `spring.` so others know that these vars are spring specific